spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisProperties.java

```diff
@@ -325,6 +326,14 @@ public void setPool(Pool pool) {
 		 */
 		private Pool pool;
 
+		/**
+		 * If shareNativeConnection is true, the pool will be used to select a connection
+		 * for blocking and tx operations only, which should not share a connection. If
+		 * native connection sharing is false, the selected connection will be used for
+		 * all operations
+		 */
+		private Boolean shareNativeConnection;
```

Comment:
We should make up our mind about a default if that makes sense. If it does, it should be reflected here (and tested).

Reply:
the default value of  `shareNativeConnection` is true.  This default value is located in `LettuceConnectionFactory` but we also could the same default value in `RedisProperties`